lib/response.js

```diff
@@ -478,16 +478,19 @@ module.exports = {
 
   /**
    * Checks if the request is writable.
-   * Tests for the existence of the socket
-   * as node sometimes does not set it.
    *
    * @return {Boolean}
    * @api private
    */
 
   get writable() {
+    // can't write any more after response finished
```

Comment:
In case anyone is wondering, this change seems fine to me, and of course checking the flag really doesn't quality to have it's own module. FWIW, the following I found to work on all Node.js versions as far back as 0.8: https://github.com/jshttp/on-finished/blob/v2.3.0/index.js#L70

It is literally the same code that is in here now, but as a single line :)